fe/fe-core/src/main/java/org/apache/doris/blockrule/SqlBlockRuleMgr.java

```diff
@@ -90,6 +91,21 @@ public List<SqlBlockRule> getSqlBlockRule(ShowSqlBlockRuleStmt stmt) throws Anal
         return Lists.newArrayList(nameToSqlBlockRuleMap.values());
     }
 
+    /**
+     * Get SqlBlockRule by show command.
+     **/
+    public List<SqlBlockRule> getSqlBlockRule(ShowSqlBlockRuleCommand cmd) throws AnalysisException {
```

Comment:
better to use String as parameter and change the old code to call the new function to remove duplicate code
```suggestion
    public List<SqlBlockRule> getSqlBlockRule(String ruleName) throws AnalysisException {
```